File: Examen Parcial/P3_20221473/TransitSoft/TransitSoftDomain/Vehiculo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TransitSoftDomain
{
    public class Vehiculo
    {
        public int VehiculoId { get; set; }
        public string Placa { get; set; }
        public string Marca { get; set; }
        public string Modelo { get; set; }
        public int Anho { get; set; }

        public Vehiculo(string placa, string marca, string modelo, int anho)
        {

            Placa = placa;
            Marca = marca;
            Modelo = modelo;
            Anho = anho;
        }
        public Vehiculo()
        {
            VehiculoId = 0;
            Placa = string.Empty;
            Marca = string.Empty;
            Modelo = string.Empty;
            Anho = 0;
        }
        public override string ToString()
        {
            return $"{Marca} {Modelo} - Placa: {Placa} - Año: {Anho}";
        }
    }
}

File: Examen Parcial/proyectostrategy/src/main/java/pe/edu/pucp/softinv/proyectostrategy/modelo/Vehiculo.java
package pe.edu.pucp.softinv.proyectostrategy.modelo;

public class Vehiculo {
    private Integer vehiculoId;
    private String placa;
    private String marca;
    private String modelo;
    private Integer anho;

    public Integer getVehiculoId() {
        return vehiculoId;
    }

    public void setVehiculoId(Integer vehiculoId) {
        this.vehiculoId = vehiculoId;
    }

    public String getPlaca() {
        return placa;
    }

    public void setPlaca(String placa) {
        this.placa = placa;
    }

    public String getMarca() {
        return marca;
    }

    public void setMarca(String marca) {
        this.marca = marca;
    }

    public String getModelo() {
        return modelo;
    }

    public void setModelo(String modelo) {
        this.modelo = modelo;
    }

    public Integer getAnho() {
        return anho;
    }

    public void setAnho(Integer anho) {
        this.anho = anho;
    }
}
